nexus/reconfigurator/planning/src/blueprint_builder/zones.rs

```diff
@@ -346,6 +346,11 @@ mod tests {
                 .expect("new zone ID should be present");
         }
 
+        // TODO: FIXME: AJS - Delete use of the control sled? - We don't
+        // actually compute changes based on generation number anymore. Zones or
+        // physical disks must change for a sled to be modified. It's not clear
+        // to me that the way we use this in this test is actually useful.
+        //
         // Also call change_sled_zones without making any changes. This
         // currently bumps the generation number, but in the future might
         // become smarter and not do so (in which case this test will break).
```

Comment:
Should we make this change and make `change_sled_zones` smarter? (Happy to defer this also, but it seems like it would address your TODO?)

Reply:
I didn't make this any smarter. I don't think it matters for my todo, and is definitely outside the scope of this PR. This is especially true because I don't know how to make it smarter :)